src/components/atoms/AchievementBadge/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import './style.scss';
+import Text from '@src/components/atoms/Text';
+import AchievementIcon from '../AchievementIcon';
```

Comment:
atom을 import할 때는 한 줄로 import하면 좋을 것 같습니다.

```suggestion
import { AchievementIcon, Text } from '@src/components/atoms';
```